posts/zzz_DO_NOT_EDIT_data__packages/data__packages.qmd

```diff
@@ -0,0 +1,88 @@
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-02-14"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [SDTM, ADaM, Community, Technical]
+# Feel free to change the image
+image: "data.jpg"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_data__packages"
+library(link)
+link::auto(keep_pkg_prefix = FALSE)
+```
+
+<!--------------- post begins here ----------------->
+
+The purpose of this blog is to maintain an ongoing list of publicly available data packages, data in packages or data sources that align to CDISC standards. My hope is that this could be a resource for:
+
+* those intrepid individuals lookings to showcase new documentation, functions, packages and other tools
+* those enterprising individuals wanting to learn more about CDISC standards and exploring open-source tools.
+
+The data presented below is just a start and is presented in order of how I found them. Feel free to get in touch with me for additions or clarifications. 
+
+## pharmaversesdtm: SDTM Test Data for the 'Pharmaverse' Family of Packages
+
+A set of Study Data Tabulation Model (SDTM) datasets from the Clinical Data Interchange Standards Consortium (CDISC) pilot project used for testing and developing Analysis Data Model (ADaM) datasets inside the pharmaverse family of packages. A CDISC Pilot was conducted somewhere between 2008 and 2010. This is that Pilot data but slowly brought up to current CDISC standards.
+
+Most common SDTM datasets can be found as well as some specific disease area SDTMs that are not avaiable in the CDISC pilot datasets.
+
+Available on [CRAN](https://cloud.r-project.org/web/packages/pharmaversesdtm/index.html) and package is actively maintained on [GitHub](https://github.com/pharmaverse/pharmaversesdtm)
+
+## pharmaverseadam: ADaM Test Data for the 'Pharmaverse' Family of Packages
+
+A set of Analysis Data Model (ADaM) datasets constructed using the Study Data Tabulation Model (SDTM) datasets contained in the {pharmaversesdtm} package and the template scripts from the {admiral} family of packages. 
+
+Available on [CRAN](https://cloud.r-project.org/web/packages/pharmaverseadam/index.html) and package is actively maintained on [GitHub](https://github.com/pharmaverse/pharmaversesdtm)
+
+## admiral: ADaM in R Asset Library
+
+A toolbox for programming Clinical Data Interchange Standards Consortium (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a mandatory part of any New Drug or Biologics License Application submitted to the United States Food and Drug Administration (FDA). Analysis derivations are implemented in accordance with the "Analysis Data Model Implementation Guide.
+
+Limited datasets like `ADSL`, `ADLB` but the template scripts availabe in this package are used to create the ADaMs in {pharmaverseadam}
```

Comment:
```suggestion
Limited datasets like `ADSL`, `ADLB` are provided in {admiral}, because the template scripts available in this package are used to create the ADaMs in {pharmaverseadam}.
```